Detectors/TPC/qc/include/TPCQC/ExpertVis.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   ExpertVis.h
+/// @author Berkin Ulukutlu, berkin.ulukutlu@cern.ch
+///
+
+#ifndef AliceO2_TPC_QC_ExpertVis_H
+#define AliceO2_TPC_QC_ExpertVis_H
+
+#include <memory>
+
+// root includes
+#include "THnSparse.h"
+
+// o2 includes
+#include "DataFormatsTPC/Defs.h"
+
```

Comment:
Is this used somewhere? If it is needed in the cxx, please move it there, otherwise just remove.

Reply:
it indeed wasn't needed, I removed it now